DESIGN.md

```diff
@@ -35,7 +35,31 @@ OSM ships out-of-the-box with all necessary components to deploy a complete serv
 
 
 
-## High-level architecture
+## OSM Components & Interactions
+![OSM Components & Interactions](./docs/images/osm-components-and-interactions.png)
+
+### Containers
+When a new Pod creation is initiated, OSM's
+[MutatingWebhookConfiguration](https://github.com/openservicemesh/osm/blob/release-v0.3/charts/osm/templates/mutatingwebhook.yaml)
```

Comment:
you can link to a file in the repo using an absolute path like: `/charts/osm/templates/mutatingwebhook.yaml`

Reply:
@michelleN I'm not opposed to switching to abs paths - but I did want to pin the link to a version for which the document is written. I worry that one day the location of the file will change, the doc will lag behind and the link will be broken.
Not sure if we have a way to do both abs path along with `v0.3` in the path?

Reply:
> Not sure if we have a way to do both abs path along with v0.3 in the path?

I don't think that's possible. If we want to pin a version though, we should document to update it in the [release guide](https://github.com/openservicemesh/osm/blob/main/docs/release_guide.md).